### PR TITLE
V1.3.4 laptimes in digital gauge

### PR DESCRIPTION
**Pull request by brentpicasso**
* Lap times show correctly in digital gauges
* Corrected incorrect channel name for "Lap"
* Refactored by reducing use of kvFind
* broke out Channels into own module
* Removed redundant classes
* Cleaned up / simplified Channel types in system channel metadata

### CODE REVIEW

File: autosportlabs/racecapture/data/channels.py [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+    def __init__(self, **kwargs):
+        self.name = kwargs.get('name', '')
+        self.units = kwargs.get('units', self.units)
+        self.min = kwargs.get('min', self.min)
+        self.max = kwargs.get('max', self.max)
+        self.precision = kwargs.get('prec', self.precision)
+        self.sampleRate = kwargs.get('sampleRate', self.sampleRate)
+        self.type = kwargs.get('type', self.type)
+        
+    def fromJson(self, json):
```
Comment (ryandoherty) [inline autosportlabs/racecapture/data/channels.py]: PEP8! :)

Comment (ryandoherty) [inline autosportlabs/racecapture/data/channels.py]: Whoops, this is not a new file, nevermind.

**Comment by ryandoherty** [comment]
Looks good :+1:

**Comment by brentpicasso** [comment]
Thanks, it turned out well, actually! pretty seamless experience.

**Pull request merged**